corehq/ex-submodules/casexml/apps/case/dbaccessors/related.py

```diff
@@ -19,11 +19,14 @@ def get_indexed_case_ids(domain, case_ids):
     )]
 
 
-def get_extension_case_ids(domain, case_ids):
+def get_extension_case_ids(domain, case_ids, exclude_for_case_type=None):
```

Comment:
It's unfortunate that this parameter needs to be added to and passed through so many functions. I will try to think about it more critically if you can't come up with a solution, but can you think of a way to avoid that?

Reply:
The call chain goes like below.

`get_ush_extension_cases_to_close` -> `get_extension_chain` -> `get_extension_case_ids` ( and in couch additionally -> `get_all_reverse_indices_info` ). 

There are multiple instances of `get_extension_case_ids` each for an interface, SQL and Couch accessor which adds to the number of places that the param needs to be passed down to.

I am not able to think of a way to consolidate much of this call chain into a single method.

Reply:
I can't think of any good ideas either. Fine to proceed.